tests/unit/Dashboard.spec.ts

```diff
@@ -263,4 +263,46 @@ describe('Dashboard - Click Tests', () => {
       done()
     })
   })
+
+  it('address/director change should be disabled', done => {
+    const wrapper = shallowMount(Dashboard,
+      {
+        store,
+        vuetify,
+        computed: {
+          disableChanges () {
+            return true
+          }
+        }
+      })
+
+    Vue.nextTick(async () => {
```

Comment:
Please use `await nextTick()` instead. I guess you copied existing code but the problem with it is that the body of the nextTick executes in a different scope and you get no detail on any errors (you only get "test didn't complete within timeout").

Reply:
Updated my tests and the other tests in this file.

Reply:
Way nicer (and we'll get proper errors if anything fails). 👍